### ISSUE TITLE
Hierarchical select taxonomies

### ISSUE THREAD

**Issue by Jserraho** [CrocoBlock/suggestions#3296]
the user to be able to select the postal code, town and province. It can be configured so that if you select, for example, the province of Barcelona, ​​only the towns and postal codes of that province are shown in the fields

**Comment by bznsolution** [comment]
That's right!! I have suggested same with you "https://github.com/Crocoblock/suggestions/issues/3242".
This is important to use this function actually.

**Comment by mali78611** [comment]
Exactly!
Now, this is what I have requested as well, they have made it possible on their smart search filters but nowhere else.
Hierarchal search system should be on the forms most importantly, in this case, the province of Barcelona would show towns and postal codes of that province.
